config.toml

```diff
@@ -50,9 +50,10 @@ ignoreFiles = [ "sensu-doc-template.md" ]
         name = "Sensu Core"
         description = "The open source extensible monitoring framework."
         weight = 1
-        latest = "1.2"
```

Comment:
Also need to update static.json to ensure `/sensu-core/latest/` is redirected to the latest version

Reply:
Done 👍